app/src/main/java/com/zeapo/pwdstore/autofill/v2/AutofillService2.kt

```diff
@@ -0,0 +1,57 @@
+package com.zeapo.pwdstore.autofill.v2
+
+import android.os.Build
+import android.os.CancellationSignal
+import android.service.autofill.*
+import android.service.autofill.AutofillService
+import androidx.annotation.RequiresApi
+import android.service.autofill.FillResponse
+import com.zeapo.pwdstore.R
+import com.zeapo.pwdstore.autofill.v2.ui.AutofillFilterView
+import android.util.Log
+import com.zeapo.pwdstore.autofill.v2.AutofillUtils.Companion.getAutocompleteEntry
+
+
+@RequiresApi(Build.VERSION_CODES.O)
+class AutofillService2 : AutofillService() {
```

Comment:
Maybe call it `PasswordAutofillService` if you're worried about namespace clashes?

Reply:
Yep, its a temporary ClassName and package. I was waiting your review for the final name.